src/workerd/server/tests/python/sdk/proxy.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  async fetch(req, env) {
+    const url = new URL(req.url);
+    if (url.hostname == 'pyodide-packages.runtime-playground.workers.dev') {
+      return env.INTERNET.fetch(req);
+    } else if (url.hostname == 'example.com') {
+      return env.PYTHON.fetch(req);
+    }
+
+    throw new Error('Invalid url: ' + req.url);
```

Comment:
New URL can normalize/encode the string. It will help us a lot if we log the parsed value. 

```suggestion
    throw new Error('Invalid url: ' + url);
```

Reply:
Right now this line is req, rather than url.